MoneySplitter.Win10/Views/ShellView.xaml.cs

```diff
@@ -14,9 +14,14 @@ public ShellView()
             DataContextChanged += (s, e) => { ViewModel = DataContext as ShellViewModel; };
         }
 
-        private void ShellFrame_Loaded(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        private void OnMenuItemClick(NavigationView sender, NavigationViewItemInvokedEventArgs args)
+        {
+            ViewModel.NavigateToClikedItemMenu((string)args.InvokedItem);
+        }
+
+        private void ShellFrame_Loaded(object sender, RoutedEventArgs e)
```

Comment:
Rename this method to OnShellFrameLoaded

Reply:
rename